sig-ai/joint-inference.md

```diff
@@ -0,0 +1,383 @@
+* [Joint Inference](#joint-inference)
+   * [Motivation](#motivation)
+     * [Goals](#goals)
+     * [Non\-goals](#non-goals)
+   * [Proposal](#proposal)
+     * [Use Cases](#use-cases)
+   * [Design Details](#design-details)
+     * [CRD API Group and Version](#crd-api-group-and-version)
+     * [Joint inference CRD](#joint-inference-crd)
+     * [Joint inference type definition](#joint-inference-type-definition)
+     * [Joint inference sample](#joint-inference-sample)
+     * [Validation](#validation)
+   * [Controller Design](#controller-design)
+     * [Joint Inference Controller](#joint-inference-controller)
+     * [Downstream Controller](#downstream-controller)
+     * [Upstream Controller](#upstream-controller)
+     * [Details of api between GC(cloud) and LC(edge)](#details-of-api-between-gccloud-and-lcedge)
+   * [Workers Communication](#workers-communication)
+   
+# Joint Inference
+## Motivation
+
```

Comment:
no architecutre picture? meanwhile,  I suggest it will be better if add some workflow picture.